system/lib/wasmfs/fuzzer/parameters.h

```diff
@@ -8,6 +8,11 @@
 #pragma once
 
 namespace wasmfs {
+//
+// Global var for verbose printing.
+//
+extern bool VERBOSE;
```

Comment:
I would expect this to be set based on a command line argument, but I don't see it set anywhere.

Reply:
I pushed the change, it just wasn't committed.